css/styles.sass

```diff
@@ -0,0 +1,20 @@
+body
+  padding: 0
+  margin: 0
+  display: grid
+  grid-template-areas: "main edit"
+  grid-template-columns: 1fr 340px
+  grid-template-rows: 100vh
+  grid-gap: 4px
+
+  main
+    grid-area: main
+
+#editor.ace_editor
+  grid-area: edit
+    padding: 0
+    margin: 0
+
+  & > *
+    padding: 0
+    margin: 0
```

Comment:
I'm not familiar with Ace, but I did notice what you mentioned regarding the issues with global styles. Would it be possible to further define the styles in the editor here by ID, or would that not work with Ace?

Reply:
I was trying to do that but my time was running low so I decided to postpone that research. I'll definitely investigate further.